lib/engines/content_block_manager/app/views/content_block_manager/content_block/documents/show.html.erb

```diff
@@ -46,31 +46,42 @@
   </div>
 </div>
 
-<% @schema.subschemas.each do |subschema| %>
```

Comment:
are we happy at least initially to just rely on the count of subschemas, or would it be preferable to define in our config?